test/multiverse/suites/redis/redis_instrumentation_test.rb

```diff
@@ -13,7 +13,7 @@ class NewRelic::Agent::Instrumentation::RedisInstrumentationTest < Minitest::Tes
     def after_setup
       super
       # Default timeout is 5 secs; a flushall takes longer on a busy box (i.e. CI)
-      @redis = Redis.new(:host => redis_host, :timeout => 25)
+      @redis ||= Redis.new(timeout: 25)
```

Comment:
Would you please explain the removals of the `redis_host` calls? Also, have ALL of the calls been removed, allowing us to remove the method definition itself now?

Reply:
While running the tests locally and across multiple versions with `redis_host`, both @hannahramadan and I ran into the following error: 

```
Redis::CannotConnectError: getaddrinfo: nodename nor servname provided, or not known
```

We had docker running Redis and this consistently occurred. Removing the `host: redis_host` argument resolved these problems and did not impact the CI, so we deemed this argument unnecessary. It has been removed from all locations, so the method definition can be removed now too. Thanks for noticing that! 

Resolved in: d49ef0fd39fc497c9bed9f0a9273c74e177e1d77

Reply:
My bad, it looks like `redis_host` is still used so that metric names can be dynamically generated based on where the tests are running. 

Reverted in: 0d8f978da144f2f8ff5bde963b14117a5d18a5cb

Reply:
Cool. Currently we just do "are we in Docker?" very generically. I could see us needing to be more specific in future, such as "are we in Docker AND in an automated CI environment and not just a desktop based Docker system?"

Reply:
So true! Good point. Do you think there's an appropriate action to take now to help future devs?